Assets/HoloToolkit/Input/Scripts/InputManager.cs

```diff
@@ -727,6 +726,6 @@ public void RaiseDictationError(IInputSource source, uint sourceId, string dicta
         }
 
         #endregion // Dictation Events
-
+#endif
```

Comment:
Usage of `RaiseDictationError`, `RaiseDictationHypothesis`, `RaiseDictationComplete` and `RaiseDictationResult` also have to be wrapped like this.
Since I didn't wrap these before I also didn't realize they have to be wrapped where they are used in the `DictationInputManager`. Result and Complete are already wrapped so it just has to be extended a little bit to include Hypothesis and Error.

Getting 2 errors for those now: 
```
Assets/HoloToolkit/Input/Scripts/Microphone/DictationInputManager.cs(248,35): error CS1061: Type `HoloToolkit.Unity.InputModule.InputManager' does not contain a definition for `RaiseDictationError' and no extension method `RaiseDictationError' of type `HoloToolkit.Unity.InputModule.InputManager' could be found. Are you missing an assembly reference?
Assets/HoloToolkit/Input/Scripts/Microphone/DictationInputManager.cs(200,35): error CS1061: Type `HoloToolkit.Unity.InputModule.InputManager' does not contain a definition for `RaiseDictationHypothesis' and no extension method `RaiseDictationHypothesis' of type `HoloToolkit.Unity.InputModule.InputManager' could be found. Are you missing an assembly reference?
```